zerver/webhooks/beeminder/doc.md

```diff
@@ -0,0 +1,29 @@
+Get Beeminder notifications in zulip whenever you're going to derail from your goal!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the url above where `api_key` is api key of \
+your Zulip bot and `email` is your zulip email.
+
+* When creating a new goal in beeminder you can check `Make private` option to let \
+bot send you PM otherwise bot will publish in stream.
+
+![](/static/images/integrations/beeminder/001.png)
+
+* Copy the above url and paste it in `WEBHOOK` input in `reminders` \
+setting of your beeminder account.
+
```

Comment:
Again,

* Replace `url` with `URL`
* Change it to `paste it in the `WEBHOOK` input option in the `reminders` setting of your Beeminder account.`
* Also, do not use a `\` for breaking the line.